Documentation/guides/Extending LiveCode.md

```diff
@@ -614,6 +614,56 @@ support/icon.png // 20x40 png image
 support/icon@extra-high.png // 40x80 png image
 ```
 
+## The Extension Store
+
+Once you have created and packaged your extension you can make it available to the LiveCode Community via the [Extension Store](https://livecode.com/products/extensions), in the [widgets](https://livecode.com/products/widgets/) section.
+
+### Registering as an Extension Developer
+
+The first step is to [register](https://livecode.com/account/developer/register) as an Extension Developer.
+
+- Log in to your LiveCode Account
+- Select the Register option under the Developer section
+
+![enter image description here](images/extensions-developer_id.png)
+
+LiveCode Extensions must have unique identifiers. These identifiers use reverse domain notation.
+
+You can sign up on this page for a LiveCode Community developer ID. Having a developer ID will allow you to use extension identifiers of the form: community.livecode.developerID.extensionName
+
+Choose the Developer ID you want to use here, if your chosen ID is already in use you will be asked to choose a different one.
+
+### Uploading Extensions to the Store
+
+To upload a packaged extension to the store log in to your LiveCode account and go to the [Extensions](https://livecode.com/account/developer/extensions) section of the Developer area.
+
+Any extensions you have uploaded will be shown here, and you have the option to add new extensions.
+
+- Click the "Add New Extension" button (if available)
+- Drag your packaged extension (.lce) file into the upload area
+
+![enter image description here](images/extensions-upload.png)
+
+- When the extension has uploaded it will appear in your list
```

Comment:
I suggest moving the final bullet point above the image, so that it isn't left sitting by itself underneath the image at the end of the section.